src/main/java/com/doctusoft/guava/converter/CommaSeparatedSortedSetConverter.java

```diff
@@ -0,0 +1,36 @@
+package com.doctusoft.guava.converter;
+
+import com.google.common.base.Converter;
+import com.google.common.base.Joiner;
+import com.google.common.base.Splitter;
+import com.google.common.collect.ImmutableSortedSet;
+import com.google.common.collect.Iterables;
+
+import java.io.Serializable;
+import java.util.*;
+
+import static java.util.Objects.*;
+
+public class CommaSeparatedSortedSetConverter<T extends Comparable<? super T>> extends Converter<Set<T>, String> implements Serializable {
+    
+    private final Converter<String, T> elementConverter;
+    
+    public CommaSeparatedSortedSetConverter(Converter<String, T> elementConverter) {
+        this.elementConverter = requireNonNull(elementConverter, "elementConverter");
+    }
+    
+    protected Set<T> doBackward(String string) {
+        TreeSet<T> set = new TreeSet<>();
+        Iterables.transform(SPLITTER.split(string), elementConverter).forEach(set::add);
+        return set;
+    }
+    
+    protected String doForward(Set<T> set) { return JOINER.join(ImmutableSortedSet.copyOf(set)); }
```

Comment:
I checked only that the doBackward creates a sorted set regardless of the order of the input.
It makes sense to store an ordered String.